src/lib/markbind/src/parser.js

```diff
@@ -522,25 +519,16 @@ class Parser {
         xmlMode: true,
         decodeEntities: false,
       });
-      // Read files
-      fs.readFile(file, (err, data) => {
-        if (err) {
-          reject(err);
-          return;
-        }
-        const fileExt = utils.getExt(file);
-        if (utils.isMarkdownFileExt(fileExt)) {
-          const inputData = md.render(data.toString());
-          context.source = 'md';
-          parser.parseComplete(inputData);
-        } else if (fileExt === 'html') {
-          context.source = 'html';
-          parser.parseComplete(data);
-        } else {
-          const error = new Error(`Unsupported File Extension: '${fileExt}'`);
-          reject(error);
-        }
-      });
+      const fileExt = utils.getExt(filePath);
+      if (utils.isMarkdownFileExt(fileExt)) {
+        const inputData = md.render(content);
```

Comment:
One tiny nit:
`inputData` seems strange now that we are no longer reading from a file.
How about `renderedContent`?